Server/sprawl_calculator/wup_calculator/tasks.py

```diff
@@ -0,0 +1,70 @@
+from celery import Celery
```

Comment:
Thank you for this, This file needs to be updated / removed. My recommendation is that we just focus on the async first and then rebase and then work on this PR.